docs/config/README.md

```diff
@@ -1314,6 +1314,7 @@ current directory.
 | `ahead`      | `"⇡"`                                         | The format of `ahead`               |
 | `behind`     | `"⇣"`                                         | The format of `behind`              |
 | `diverged`   | `"⇕"`                                         | The format of `diverged`            |
+| `uptodate`   | `"✓"`                                         | The format of `uptodate`            |
```

Comment:
I think it's better to default to an empty string to avoid changing the existing output.

Reply:
In the docstrings in `modules/git_status.rs` (lines 13-26), should I remove the mention of `uptodate` there since it defaults to nothing, or still leave it there with an empty string as its 'default symbol'? Thanks for your review!

Reply:
Please leave it there with an empty string.